api/src/main/java/org/openmrs/module/sync/SyncUtil.java

```diff
@@ -1387,4 +1388,8 @@ public static String formatTransactionStatus(Transaction tx) {
 		}
 		return "TX STATUS UNKNOWN";
 	}
+
+	public static int daysBetween(Date old, Date later){
+		return (int)( (later.getTime() - old.getTime()) / (1000 * 60 * 60 * 24));

```

Comment:
You could also convert (1000 * 60 * 60 * 24) to a method with a clear name.
I know that it is converting a day right?

Reply:
I will put the constants in the method itself.